experiments/optimal-core-allocation/README.md

```diff
@@ -2,50 +2,49 @@
 
 ## Rationale
 
-This experiment allows you to find the optimal core and thread workers configuration that delivers the best performance 
-in terms of throughput and guaranteeing latency for sensitive workloads.
+This experiment allows you to find the optimal core and worker threads configuration that delivers the best performance 
+in terms of throughput and guaranteeing response time for latency sensitive workloads.
 
-This example experiment uses memcached as the latency sensitive workload and tries to find optimal number of memcached worker threads (typically set with  the `-t` flag for `memcached`).
+The experiment helps user to find optimal number of memcached worker threads (typically set with  the `-t` flag for `memcached`) by running memcached as the latency sensitive workload with various worker threads configuration.
 
-#### Environment prerequisites
+### Environment prerequisites
 
-1. Download and install in $PATH `optimal-core-allocation` experiment binary from [releases page](https://github.com/intelsdi-x/swan/releases) (available inside `swan.tar.gz`) along with all required [snap](https://github.com/intelsdi-x/snap) plugins.
-1. [mutilate](https://github.com/leverich/mutilate) load generator preinstalled on load generator nodes (flags: `-experiment_mutilate_master_address`, `-experiment_mutilate_agent_addresses`).
+1. `optimal-core-allocation` experiment binary available in `$PATH`. The binary can be downloaded from [releases page](https://github.com/intelsdi-x/swan/releases) (available inside `swan.tar.gz`) along with all required [Snap](https://github.com/intelsdi-x/snap) plugins.
```

Comment:
Not all snap plugins are in package. This experiment can use USE collector that is not available in this package.
Could you please point to `https://github.com/intelsdi-x/snap-plugin-collector-use/releases/download/1/snap-plugin-collector-use_linux_x86_64` to download it?